Makefile

```
@@ -281,13 +281,23 @@ api-fix-versions:
 api-doc:
 	./contrib/apiage.py --mode=write-doc
 
-api-report-updates: $(RESULTS_DIR)
+api-check-updates: $(RESULTS_DIR)
 	./contrib/apiage.py --mode=find-updates \
 		--current-tag="$$(git describe --tags --abbrev=0)" \
 		> $(RESULTS_DIR)/updates-found.json
+
+api-report-updates:
+	$(MAKE) api-check-updates
 	./contrib/apiage.py --mode=updates-to-markdown \
 		< $(RESULTS_DIR)/updates-found.json > $(RESULTS_DIR)/updates-found.md
 
+api-report-issuetemplate:
+	$(MAKE) api-check-updates
```

Same comment as above.
```suggestion
api-report-issuetemplate: api-check-updates
```

---

Fixed